Automattic/woocommerce-payments
# Update Qualitative Feedback note to have more efficient sql query

Fixes #3693 

#### Changes proposed in this Pull Request

Update the sql query to no longer use `count` to make it more efficient, and also make it work if there are any WooPayments related transactions, not just stored cards.

#### Testing instructions

* Navigate to _WCPay Dev_ tools.
* Use the _Delete all WCPay notes from WC inbox_ action on the right hand side of the page.
* Navigate to WooCommerce > Home.
* Confirm the note below appears with a low time, such as a few seconds or a minute, based on how long it's been since your last step.
  * Note, this does require at least one transaction in your store.

![Screenshot 2023-12-04 at 11 02 45 AM](https://github.com/Automattic/woocommerce-payments/assets/4634416/6b9ba6f8-2c22-4cbc-b85a-e9f5afd70585)


-------------------


**Post merge**
